packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -243,8 +257,8 @@ function NoteEditor(props: Props, ref: any) {
 		}
 	`;
 
-	// const css = useCss(props.themeId);
-	const html = useHtml('');
+	const css = useCSS(props.themeId);
```

Comment:
You don't need an effect. When a variable value is simply derived from another variable, you can use useMemo. Just give props.themeId as dependency.

Reply:
That should be changed!